src/PJ_bertin1953.c

```diff
@@ -0,0 +1,109 @@
+/*
+  Created by Jacques Bertin in 1953, this projection was the go-to choice
+  of the French cartographic school when they wished to represent phenomena
+  on a global scale.
+  
+  Formula designed by Philippe Rivière, 2017.
+  https://visionscarto.net/bertin-projection-1953
+
+  Port to PROJ by Philippe Rivière, 21 September 2018
+*/
+
+#define PJ_LIB__
+
+#include <errno.h>
+#include <math.h>
+
+#include "proj.h"
+#include "projects.h"
+
+PROJ_HEAD(bertin1953, "Bertin 1953")
+    "\n\tMisc Sph no inv.";
+
+struct pj_opaque {
+    double w;
+    double m, rm;
+    double cosDeltaPhi, sinDeltaPhi, cosDeltaGamma, sinDeltaGamma, deltaLambda;
+};
+
+
+static XY s_forward (LP lp, PJ *P) {
+    XY xy = {0.0,0.0};
+    struct pj_opaque *Q = P->opaque;
+
+    /* Projection constants */
+    double fu = 1.4, k = 12., w = 1.68;
+
+    /* Aliases */
+    double lambda = lp.lam, phi = lp.phi;
+    
+    /* Variable */
```

Comment:
It's not really necessary to explain that this is a variable - C-programmers will be familiar the syntax :-)